### PR TITLE
Make it easier to get help

### LINKED ISSUE #21206: Admin troubleshooting page changes
The app-side portion of the "Make it easier to get help" project. Mostly just a few minor tweaks.

**Links**
- product doc: https://www.notion.so/metabase/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977

**Implementation Plan**
- Make the "Help" option in the top right gear icon menu dynamic per the product doc -- https://www.notion.so/metabase/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977#eb32a7e4d4e34146bcb6ecad39d9e6a8
- Update the "Get help" box per the designs  -- https://www.notion.so/metabase/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977#974bea7244fc4fa49fd5436e4f821b98

### PR DESCRIPTION
**Pull request by WiNloSt**
Fixes #21206

### CODE REVIEW

File: frontend/src/metabase/nav/components/ProfileLink.jsx [hunk 1]
```diff
@@ -63,7 +63,9 @@ export default class ProfileLink extends Component {
       {
         title: t`Help`,
         icon: null,
-        link: MetabaseSettings.docsUrl(),
+        link: MetabaseSettings.isPaidPlan()
+          ? "https://www.metabase.com/help-premium/"
+          : "https://www.metabase.com/help/",
```
Comment (WiNloSt) [inline frontend/src/metabase/nav/components/ProfileLink.jsx:68]: I was tempted to move this inside MetabaseSettings, so we could do something like `MetabaseSettings.helpUrl()`. But I'm not sure if we'll do this in a lot of places, and I don't want to pollute the Settings file yet.

File: frontend/src/metabase/admin/tasks/containers/Help.jsx [hunk 2]
```diff
@@ -58,13 +58,6 @@ function githubIssueLink(bugReportDetails) {
   );
 }
 
-function discourseLink(bugReportDetails) {
```
Comment (WiNloSt) [inline frontend/src/metabase/admin/tasks/containers/Help.jsx:61]: Update the help page according to the design https://www.notion.so/metabase/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977#974bea7244fc4fa49fd5436e4f821b98

File: frontend/src/metabase/nav/components/ProfileLink.jsx [hunk 3]
```diff
@@ -63,7 +63,11 @@ export default class ProfileLink extends Component {
       {
         title: t`Help`,
         icon: null,
-        link: MetabaseSettings.docsUrl(),
+        link:
+          isAdmin && MetabaseSettings.isPaidPlan()
```
Comment (WiNloSt) [inline frontend/src/metabase/nav/components/ProfileLink.jsx:67]: The logic is based on a comment here (Under `UTM Trackings` at `Non-admin` on the first row, I can't link to comment inside a table for some reason) https://www.notion.so/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977?d=8acba01bac36481c99da1b786f4f57e6

File: frontend/src/metabase/nav/components/ProfileLink.jsx (hunk 1 above)
Comment (WiNloSt) [reply frontend/src/metabase/nav/components/ProfileLink.jsx]: I just updated it.

**Review by daltojohnso — APPROVED** [review]
👍

**Comment by WiNloSt** [comment]
@brunobergher and you have a quick look to see if this looks ok now?

**Review by brunobergher — COMMENTED** [review]
The logic and links all look good to me!

**Comment by WiNloSt** [comment]
@flamber Do you think this should be backported?

**Pull request merged**